db/migrate/20171024074942_convert_autograded_text_response_answers_to_plaintext.rb

```diff
@@ -0,0 +1,23 @@
+class ConvertAutogradedTextResponseAnswersToPlaintext < ActiveRecord::Migration[5.0]
+  # For autograded text response questions, a plaintext textarea input box is shown instead of
+  # the rich text editor.
+  #
+  # Convert existing answers to autograded questions from HTML to plaintext.
+  def up
+    auto_gradable_tr_question_ids = Course::Assessment::Question::TextResponse.includes(:solutions).
+                                    select(&:auto_gradable?).map(&:acting_as).map(&:id)
+
+    Course::Assessment::Answer.where(question_id: auto_gradable_tr_question_ids).includes(:actable).
+      find_each do |ans|
+      answer_text = ans.actable.answer_text
+      next if answer_text.empty?
+      ans.actable.answer_text = Sanitize.fragment(answer_text).strip.encode(universal_newline: true)
+      ans.actable.save!(validate: false)
```

Comment:
think it's better to use`ans.actable.update_column(:answer_text, new_value)`, so that timestamps are not changed. `save` will change the timestamp of the answer.